app/assets/javascripts/foreman_virt_who_configure/config_edit.js

```diff
@@ -26,6 +37,9 @@ $(document).ready(function () {
     });
   });
 
+
   virt_who_update_listing_mode();
+  virt_who_update_hypervisor_fields();
   $('#foreman_virt_who_configure_config_listing_mode').change(virt_who_update_listing_mode);
+  $('#foreman_virt_who_configure_config_hypervisor_type').change(virt_who_update_hypervisor_fields);
```

Comment:
There's an onchange callback on the line `31` already. Could you please either extract it into a function and put both `.change` calls next to each other or put the functions into the anonymous function on the line `31`? Just to improve readability.